fixtures/WrongCall.php

```diff
@@ -5,7 +5,7 @@
 use PHPUnit\Framework\TestCase;
 use Prophecy\PhpUnit\ProphecyTrait;
 
-class Error extends TestCase
+class WrongCall extends TestCase
```

Comment:
which possible naming collisions ? The namespace `Prophecy\PhpUnit\Tests\Fixtures` is fully ours.

Reply:
I was getting a "Cannot find class `Error`" with that, and the simple renaming made it go away.